src/reducers/eventListeners.test.js

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import * as reducerCreators from './reducerCreators';
+import * as selectorCreators from './selectorCreators';
+import createReducer, * as selectors from './eventListeners';
+
+const name = 'pipeline resource name';
```

Comment:
copy & paste nit: should be `event listener name` 😄

Reply:
Done 😆